### PR TITLE
Fix testerina_function_mocks.out

### PR DESCRIPTION
**Pull request by Ibaqu**
## Purpose
Function mocking currently only works when the tests are within a ballerina project in the test module. This needed to be explicitly mentioned in the .out file. 

This will be changed when mocking for single test files is supported. 

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: examples/testerina-function-mocks/testerina_function_mocks.out [hunk 1]
```diff
@@ -2,14 +2,19 @@
 # `.bal` file, and execute the `ballerina test` command.
 # Note that you need to have the ballerina-tools distribution
 # installed in order to run this command.
-$ ballerina test testerina_function_mocks.bal
-Compiling tests
-    testerina_function_mocks.bal
+# Function mocking is not supported for testing single bal files
```
Comment (praneesha) [inline examples/testerina-function-mocks/testerina_function_mocks.out]:
```suggestion
# Function mocking is not supported for testing single BAL files.
```

File: examples/testerina-function-mocks/testerina_function_mocks.out [hunk 2]
```diff
@@ -2,14 +2,19 @@
...
+$ ballerina test test_module
+Compiling source
```
Comment (praneesha) [inline examples/testerina-function-mocks/testerina_function_mocks.out]:
```suggestion
Compiling the source.
```

File: examples/testerina-function-mocks/testerina_function_mocks.out [hunk 3]
```diff
@@ -2,14 +2,19 @@
...
+    ballerinatest/test_module:0.1.0
+
+Creating balos
```
Comment (praneesha) [inline examples/testerina-function-mocks/testerina_function_mocks.out]:
```suggestion
Creating the BALOs.
```

File: examples/testerina-function-mocks/testerina_function_mocks.out [hunk 4]
```diff
@@ -2,14 +2,19 @@
...
+    target/balo/test_module-2020r1-any-0.1.0.balo
 
 Running tests
```
Comment (praneesha) [inline examples/testerina-function-mocks/testerina_function_mocks.out]:
```suggestion
Running the tests.
```

File: examples/testerina-function-mocks/testerina_function_mocks.out (hunk 1 above)
Comment (Ibaqu) [reply examples/testerina-function-mocks/testerina_function_mocks.out]: Done

File: examples/testerina-function-mocks/testerina_function_mocks.out (hunk 3 above)
Comment (Ibaqu) [reply examples/testerina-function-mocks/testerina_function_mocks.out]: This is part of the output

File: examples/testerina-function-mocks/testerina_function_mocks.out (hunk 4 above)
Comment (Ibaqu) [reply examples/testerina-function-mocks/testerina_function_mocks.out]: This is part of the output

File: examples/testerina-function-mocks/testerina_function_mocks.out (hunk 3 above)
Comment (Ibaqu) [reply examples/testerina-function-mocks/testerina_function_mocks.out]: This is part of the output

**Comment by Ibaqu** [comment]
https://github.com/ballerina-platform/ballerina-lang/commit/4701b2c444230f890958eaab01db66cc5c036e77

This has been merged in this commit

**Pull request closed without merging**